migration/identity-migration/src/main/java/io/camunda/migration/identity/config/saas/StaticEntities.java

```diff
@@ -152,4 +158,113 @@ public class StaticEntities {
+          new CreateAuthorizationRequest(
+              ZEEBE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.RESOURCE,
+              Set.of(
+                  PermissionType.CREATE,
+                  PermissionType.DELETE_FORM,
+                  PermissionType.DELETE_PROCESS,
+                  PermissionType.DELETE_DRD,
+                  PermissionType.DELETE_RESOURCE)),
+          new CreateAuthorizationRequest(
+              ZEEBE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.PROCESS_DEFINITION,
+              Set.of(
+                  PermissionType.UPDATE_PROCESS_INSTANCE,
+                  PermissionType.UPDATE_USER_TASK,
+                  PermissionType.CREATE_PROCESS_INSTANCE,
+                  PermissionType.DELETE_PROCESS_INSTANCE)),
+          new CreateAuthorizationRequest(
+              ZEEBE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.DECISION_DEFINITION,
+              Set.of(
+                  PermissionType.CREATE_DECISION_INSTANCE,
+                  PermissionType.DELETE_DECISION_INSTANCE)),
+          new CreateAuthorizationRequest(
+              ZEEBE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.DECISION_REQUIREMENTS_DEFINITION,
+              Set.of(PermissionType.UPDATE, PermissionType.DELETE)),
+          // OPERATE
+          new CreateAuthorizationRequest(
+              OPERATE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.MESSAGE,
+              Set.of(PermissionType.READ)),
+          new CreateAuthorizationRequest(
+              OPERATE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.BATCH_OPERATION,
+              Set.of(PermissionType.READ, PermissionType.CREATE, PermissionType.UPDATE)),
+          new CreateAuthorizationRequest(
+              OPERATE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.RESOURCE,
+              Set.of(PermissionType.READ)),
+          new CreateAuthorizationRequest(
+              OPERATE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.PROCESS_DEFINITION,
+              Set.of(
+                  PermissionType.READ_PROCESS_DEFINITION,
+                  PermissionType.READ_PROCESS_INSTANCE,
+                  PermissionType.DELETE_PROCESS_INSTANCE)),
+          new CreateAuthorizationRequest(
+              OPERATE_CLIENT_ID,
+              AuthorizationOwnerType.CLIENT,
+              "*",
+              AuthorizationResourceType.DECISION_DEFINITION,
+              Set.of(
```

Comment:
💭 for process we had delete_process_instance for decisions not? let's align https://docs.google.com/document/d/1XqG9Y9SXvakwcJ7ZheE8lgd6X0PMWSuyH4IbYTzLYeE/edit?pli=1&disco=AAABiA5afxw

Reply:
I will follow-up on this here https://docs.google.com/document/d/1XqG9Y9SXvakwcJ7ZheE8lgd6X0PMWSuyH4IbYTzLYeE/edit?disco=AAABiA5afxw&pli=1